### ISSUE TITLE
Parser don't work in IE11 and Microsoft Edge

### ISSUE THREAD

**Issue by ArtKuz** [WrinklyNinja/jed-gettext-parser#9]
Hello.
The problem is that TextDecoder is not supported by IE. http://caniuse.com/#feat=textencoder
I think that the problem can be solved by updating text-encoding in dependencies.

**Comment by WrinklyNinja** [comment]
Looks like the Sauce Labs badge in the readme is broken too, so I need to update that (and now that a caniuse link exists for TextDecoder, link to it).

**Comment by ArtKuz** [comment]
I experimented a bit and found out that the problem is not in the version of TextDecoder. Now I want to do pull request.

**Comment by WrinklyNinja** [comment]
Closing this as the updated Sauce Labs tests show the library works in Edge and IE 11.

**Comment by ArtKuz** [comment]
I looked at your updates and updated this package at home. But I ran into the same problem.
In Windows 10 in IE 11 and Microsoft Edge I get error on string 84:
```js
var decoder = new TextDecoder();
```
and after get exeption from string 164:
```js
throw new Error("The encoding label provided ('" + this._encoding + "') is invalid.");
```
Error in string 162: 
```js
var decoder = new TextDecoder(this._encoding);
```
In strings 20-24 I always get 
```js
var TextDecoder = window.TextDecoder; // undefined
```
My fix resolve this problems.
What do you think?
I reopen pr.

**Comment by shketko** [comment]
Is it still doesn't work in IE ?
I get an error `TypeError: object doesn’t support property or method`

IE 11

**Comment by shketko** [comment]
@Ortham, this patch really fix this problem https://github.com/Ortham/jed-gettext-parser/pull/10/commits/a5e3a3a8e3bb91ea5ac33011e48ee783ba459685

Can you merge it, please?

**Comment by Ortham** [comment]
I don't see any issues running the tests in IE:

![Screenshot (78)](https://user-images.githubusercontent.com/2430738/76897413-9f440780-688b-11ea-9aa9-a3491ab9aeeb.png)

The tests load the `TextDecoder` polyfill before loading `jed-gettext-parser`, which does not load any browser polyfills. This will not change: if you need the polyfill, make sure you load it.

**Comment by shketko** [comment]
Hmm... I'm using npm (https://www.npmjs.com/package/jed-gettext-parser), but version which I try is a very old. That's the reason. Can you update your package in npm?

**Comment by Ortham** [comment]
I've just released v1.0.1.
